clang/lib/Format/FormatToken.cpp

```diff
@@ -66,13 +72,17 @@ bool FormatToken::isSimpleTypeSpecifier() const {
   case tok::kw_decltype:
   case tok::kw__Atomic:
     return true;
+  case tok::identifier:
+    return IsCpp && std::binary_search(CppNonKeywordTypes.begin(),
+                                       CppNonKeywordTypes.end(), TokenText);
```

Comment:
Should this really be C++-only? All of the types in your list are standard library header types in C too, except for `byte` (which, as noted above, probably shouldn't be there).

Reply:
There’s no C per se in clang-format. `IsCpp` by clang-format convention means C/C++/Objective-C/
Objective-C++.